lib/ansible/modules/cloud/azure/azure_rm_cosmosdbaccount.py

```diff
@@ -0,0 +1,544 @@
+version_added: "2.8"
+short_description: Manage Azure Database Account instance.
+description:
+    - Create, update and delete instance of Azure Database Account.
+
+options:
+    resource_group:
+        description:
+            - Name of an Azure resource group.
+        required: True
+    name:
+        description:
+            - Cosmos DB database account name.
+        required: True
+    location:
+        description:
+            - The location of the resource group to which the resource belongs.
+            - Required when C(state) is I(present).
+    kind:
+        description:
+            - Indicates the type of database account. This can only be set at database account creation.
+        choices:
+            - 'global_document_db'
+            - 'mongo_db'
+            - 'parse'
+    consistency_policy:
+        description:
+            - The consistency policy for the Cosmos DB account.
+        suboptions:
+            default_consistency_level:
+                description:
+                    - The default consistency level and configuration settings of the Cosmos DB account.
+                    - Required when C(state) is I(present).
+                choices:
+                    - 'eventual'
+                    - 'session'
+                    - 'bounded_staleness'
+                    - 'strong'
+                    - 'consistent_prefix'
+            max_staleness_prefix:
+                description:
+                    - "When used with the Bounded Staleness consistency level, this value represents the number of stale requests tolerated. Accepted range
+                       for this value is 1 - 2,147,483,647. Required when I(default_consistency_policy) is set to C(bounded_staleness)."
+                type: int
+            max_interval_in_seconds:
+                description:
+                    - "When used with the Bounded Staleness consistency level, this value represents the time amount of staleness (in seconds) tolerated.
+                       Accepted range for this value is 5 - 86400. Required when I(default_consistency_policy) is set to C(bounded_staleness)."
+                type: int
+    geo_rep_locations:
+        description:
+            - An array that contains the georeplication locations enabled for the Cosmos DB account.
+            - Required when C(state) is I(present).
+        type: list
+        suboptions:
+            name:
+                description:
+                    - The name of the region.
+            failover_priority:
+                description:
+                    - "The failover priority of the region. A failover priority of 0 indicates a write region. The maximum value for a failover priority =
+                       (total number of regions - 1). Failover priority values must be unique for each of the regions in which the database account exists."
+                type: int
+    database_account_offer_type:
+        description:
```

Comment:
should probably have `choices` specified if possible

Reply:
I think right now the only value is "Standard", maybe something else will be introduced in the future.